### PR TITLE
Integrate Swagger and add migration files

### PR DESCRIPTION
**Pull request by lvthang1606**
- [x] Add an empty API (eg GET api/v1/books)
- [x] Add library
- [x] Check the Swagger UI and make sure it works correctly
- [x] Add migration files to create the database tables following the requirement
Screen Shot 2023-03-02 at 4 44 13 PM

### CODE REVIEW

File: src/main/java/com/thangle/persistence/book/BookStore.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import static com.thangle.persistence.book.BookEntityMapper.toBooks;
+
+@Repository
+@RequiredArgsConstructor
+public class BookStore {
+
+    private final BookRepository bookRepository;
+
+    public List<Book> findAll() {
+        List<Book> books = toBooks(toList(bookRepository.findAll()));
```
Comment (nguyenvanman) [inline src/main/java/com/thangle/persistence/book/BookStore.java]:
```suggestion
        return toBooks(toList(bookRepository.findAll()));
```

File: src/main/java/com/thangle/api/book/BookController.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import java.util.List;
+
+import static com.thangle.api.book.BookDTOMapper.toBookDTOs;
+
+@RestController
+@RequestMapping("/api/v1/books")
+@RequiredArgsConstructor
+public class BookController {
+
+    private final BookService bookService;
```
Comment (nguyenvanman) [inline src/main/java/com/thangle/api/book/BookController.java:20]: Please add unit-tests for controller, service, store and mappers

File: src/main/java/com/thangle/persistence/book/BookEntity.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import javax.persistence.Id;
+import java.util.UUID;
+
+@Table(value = "books")
+@Getter
+@SuperBuilder
+@NoArgsConstructor
+@AllArgsConstructor
+public class BookEntity {
+    public static final String ID_FIELD = "id";
```
Comment (nguyenvanman) [inline src/main/java/com/thangle/persistence/book/BookEntity.java:19]: Redundant, right?

**Review by nguyenvanman — CHANGES_REQUESTED** [review]

File: src/main/java/com/thangle/persistence/book/BookEntity.java (hunk 3 above)
Comment (voquanghoa) [reply src/main/java/com/thangle/persistence/book/BookEntity.java:19]: Ok to keep it

**Review by nguyenvanman — APPROVED** [review]

**Pull request merged**